src/aws.js

```diff
@@ -13,7 +13,7 @@ function buildUserDataScript(githubRegistrationToken, label) {
       `echo "${config.input.preRunnerScript}" > pre-runner-script.sh`,
       'source pre-runner-script.sh',
       'export RUNNER_ALLOW_RUNASROOT=1',
-      `./config.sh --url https://github.com/${config.getGitHubApiRepoPath()} --token ${githubRegistrationToken} --labels ${label}`,
+      `./config.sh --url https://github.com/${config.getGitHubApiRepoPath()} --token ${githubRegistrationToken} --labels ${label} --runnergroup default`,
```

Comment:
I think I have the solution to prevent a job to be picket by the manager.

The label that it is randomly generated is needed: it is the only way this action can identifies the runner to destroy it when the time comes.
The docs says: the `runs-on` selects the runners that matches ALL tags.
So what you should try here is to do this:
```
... --labels ${label},worker ...
```

And configure the workflow like this:
```
...
runs-on: [self-hosted, workder]
...
```
This way we're SURE the proper worker will be picked.
What do you think?

Reply:
NICE!
Excelent idea as always! I was trying to find something about negating labels but this is way more precise.
I'll change it and then do some more tests with it